config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "works#homepage"
+  resources :works
+  resources :users
+  get "/works/:work_id/upvote", to: "votes#upvote", as: "upvote"
```

Comment:
I don't think you need all 7 restful routes for `users` - only `index` and `show` are needed for this project.